packages/scxa-tsne-widget/package.json

```diff
@@ -38,9 +38,9 @@
     "@ebi-gene-expression-group/scxa-tsne-plot": "^5.3.0-alpha.0",
     "expression-atlas-autocomplete": "^3.2.0",
     "prop-types": "^15.7.2",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "styled-components": "^5.1.1",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "react-test-renderer": "^17.0.0",
```

Comment:
`react-test-renderer` should go to `devDependencies`.

Reply:
Cool! I'll move it there and push!